src/waveform/renderers/waveformrendererendoftrack.cpp

```diff
@@ -94,9 +96,7 @@ void WaveformRendererEndOfTrack::draw(QPainter* painter,
         return;
     }
 
-    const double dPlaypos = m_waveformRenderer->getPlayPos();
-    const double remainingFrames = (1.0 - dPlaypos) * 0.5 * trackSamples;
```

Comment:
Uhh, that factor 0.5 actually meaning 2 samples per frame was really ugly. Good to get rid of it ;)